ConfigurationSystem/Client/Utilities.py

```diff
@@ -384,9 +384,9 @@ def getGridSRMs( vo, bdiiInfo = None, srmBlackList = None, unUsed = False ):
     result = getBdiiSEInfo( vo )
     if not result['OK']:
       return result
-    seBdiiDict = result['Value']
+    seBdiiDict = dict( result['Value'] )
```

Comment:
This is in the pylint related commit, but for my curiosity, what is the link with pylint here ? You are basicaly making a copy of it the result ?